crypto-first-war.md

```diff
@@ -0,0 +1,61 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | Crypto First War |
```

Comment:
What can we add to make this title more captivating?

What was the main point of the first crypto war?

For instance the title could be "Crypto First War - freedom of speech in the digital age"

Reply:
I think the first one has the biggest impact "Decoding the Drama of the First Cyber War"

Reply:
Resolved